introduction.md

```diff
@@ -10,3 +10,11 @@ Si señor!
 
 ok! (ruby)
 
+## Deadly skills
+
+* Ruby
+* Git
+* Javascript
+* HTML
+* CSS
+
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?